.github/workflows/master.yml

```diff
@@ -711,6 +711,13 @@ jobs:
         with:
           name: minikube_binaries
 
+      - name: Pre-cleanup
```

Comment:
this build step runs on GH action VMs and it will not run on the Self-hosted VMS
this pre clean up needs to be added to Two Windows, ARM Jobs,

Reply:
it's in `functional_docker_ubuntu_arm64` job and runs only on self-hosted arm64 runner as it's stated in PR name. AFAIK, we only have the issue with leftovers there. 
Do we need to fix windows machines as well?

Reply:
https://github.com/kubernetes/minikube/pull/11151/checks?check_run_id=2385428597 runs on our arm64 aws instance  as expected

Reply:
sorry I was confused, you are right, yes let please let add to windows machines as well, since they are also self hosted

Reply:
done!